src/runtime/pkg/oci/utils.go

```diff
@@ -662,6 +662,16 @@ func addHypervisorMemoryOverrides(ocispec specs.Spec, sbConfig *vc.SandboxConfig
 		return err
 	}
 
+	if err := newAnnotationConfiguration(ocispec, vcAnnotations.DefaultMaxMemory).setUintWithCheck(func(memorySz uint64) error {
+		if memorySz < vc.MinHypervisorMemory && sbConfig.HypervisorType != vc.RemoteHypervisor {
```

Comment:
Why do you exclude the remote hypervisor case here?

Reply:
CSP VMs have the PCIe topology configured, but I guess for libvirt we could use it @stevenhorsman I do not have a lot of insights into the libvirt provider, would need to talk to Steve first if that makes sens.